frontend/app/routes/protected/person-case/review.tsx

```diff
@@ -196,9 +199,89 @@ export default function Review({ loaderData, actionData, params }: Route.Compone
                   <p>{t('protected:review.choosen-file')}</p>
                 </DescriptionListItem>
               </DescriptionList>
-              <ButtonLink file="routes/protected/person-case/primary-docs.tsx" variant="link" size="lg">
+              <InlineLink file="routes/protected/person-case/primary-docs.tsx">
                 {t('protected:review.edit-primary-identity-document')}
-              </ButtonLink>
+              </InlineLink>
+            </section>
+            <section className="space-y-6">
+              <h2 className="font-lato mt-3 text-2xl font-bold">{t('protected:secondary-identity-document.page-title')}</h2>
+              <DescriptionList className="divide-y border-y">
+                <DescriptionListItem term={t('protected:secondary-identity-document.document-type.title')}>
+                  <p>
+                    {t(
+                      `protected:secondary-identity-document.document-type.options.${inPersonSINCase.secondaryDocument.documentType}` as 'protected:primary-identity-document.document-type.options.select-option',
+                    )}
+                  </p>
+                </DescriptionListItem>
+                <DescriptionListItem term={t('protected:secondary-identity-document.expiry-date.title')}>
+                  <p>{inPersonSINCase.secondaryDocument.expiryDate}</p>
+                </DescriptionListItem>
+                <DescriptionListItem term={t('protected:review.document-uploaded')}>
+                  <p>{t('protected:review.choosen-file')}</p>
+                </DescriptionListItem>
+              </DescriptionList>
+              <InlineLink file="routes/protected/person-case/secondary-doc.tsx">
+                {t('protected:review.edit-secondary-identity-document')}
+              </InlineLink>
+            </section>
+            <section className="space-y-6">
+              <h2 className="font-lato mt-3 text-2xl font-bold">{t('protected:review.sub-title-preferred-name')}</h2>
```

Comment:
Headings #2 don't need `mt-3` class name since the section are spaced from their parent element with class `space-y-10`.